addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicBridgeHandler.java

```diff
@@ -223,10 +225,14 @@ public HomematicGateway getGateway() {
     /**
      * Updates the thing for the given Homematic device.
      */
-    public void updateThing(HmDevice device) {
+    private void updateThing(HmDevice device) {
         Thing hmThing = getThingByUID(UidUtils.generateThingUID(device, getThing()));
         if (hmThing != null && hmThing.getHandler() != null) {
-            hmThing.getHandler().thingUpdated(hmThing);
+            ThingHandler thingHandler = hmThing.getHandler();
+            thingHandler.thingUpdated(hmThing);
+            for (Channel channel : hmThing.getChannels()) {
+                thingHandler.channelLinked(channel.getUID());
```

Comment:
I need this only for refreshing the values in the channels. e.g if i send the REFRESH command or in case of a lost connection and a reconnect to the gateway. Normally it's not needed.